docs/src/main/asciidoc/resteasy-reactive-migration.adoc

```diff
@@ -120,6 +120,18 @@ Quarkus uses smart defaults when determining the media type of Jakarta REST meth
 The difference between `quarkus-resteasy-reactive` and `quarkus-resteasy` is the use of `text/plain` as the default media type instead of `text/html`
 when the method returns a `String`.
 
+=== Servlets
+
+RESTEasy Reactive does **not** support servlets.
+If your project depends on servlets you have to migrate them.
+A common situation is when your application is developed with JAX-RS for a servlet-based environment.
```

Comment:
I don't think this sentence provides much value

Reply:
I removed it.